doc/devguide.md

```diff
@@ -5,6 +5,10 @@ The C# development setup is described below. The Rust development setup is descr
 
 If you want to run tests outside of the pipelines, you will need to be running linux.
 
+## Warnings
+
+- If you have upgraded the most recent API version and have backported to 1.2, make sure PR for main is merged before the PR to 1.2. This is because the pipelines for main rely on images built from 1.2.
```

Comment:
> the pipelines for main rely on images built from 1.2

I don't think that's true. Was the pipeline manually triggered? Or was this an automated run?

Reply:
It was an automated run, and I was informed by someone internal that this was occurring.

Reply:
@onalante-msft Can you dm a link to the pipeline?

Reply:
Okay, it looks like we have internal folks that use edgelet from main but images from 1.2, hence the failure. I will revise the document to indicate this.

Reply:
If we have internal folks doing stuff with internal pipelines, maybe we can just document this internally to avoid confusion....

Reply:
Yes, that sounds like a better plan. I was originally unsure of where this issue arose. I will close this PR.